### DISCUSSION TITLE
Which rule has better performance?

### DISCUSSION THREAD

**Discussion by peace2000** [uBlockOrigin/uBlock-issues#2195] [Q&A] [answered]
On this page: `https://www.is.fi/`

I want to remove class `sp-message-open` from the `html` element.

Which one of these two rules would have better performance or are they equal?

`is.fi##+js(rc, sp-message-open, , stay)`
`is.fi##+js(rc, sp-message-open, html, stay)`

i.e. how will giving / not giving the css selector affect the performance? The site doesn't have other elements than that having that class.

**Comment by gorhill** [comment, marked as answer]
I doubt it would be measurable but theoretically using `html` should be better.
